src/ert/parsing/lark_parser.py

```diff
@@ -189,11 +188,15 @@ def _tree_to_dict(
     config_file: str,
     pre_defines: Defines,
     tree: Tree[Instruction],
+    schema: Mapping[str, SchemaItem],
     site_config: Optional[Dict[str, Any]] = None,
 ) -> Mapping[str, Instruction]:
-    schema: Mapping[str, SchemaItem] = (
-        init_user_config() if site_config is not None else init_site_config()
-    )
+    if schema is None:
```

Comment:
schema is never None so this if statement can be removed.

Reply:
Fixed